### PR TITLE
[4.0]namespace administrator components com_messages

### PR DESCRIPTION
**Pull request by brianteeman**
continuing the work to namespace joomla4 one admin component at a time.

please wait for drone to successfully complete before reviewing.

### CODE REVIEW

File: administrator/components/com_messages/Table/MessageTable.php [hunk 1]
```diff
@@ -53,34 +55,34 @@ public function check()
 		}
 
 		// Check the to and from users.
-		$user = new \JUser($this->user_id_from);
+		$user = new User($this->user_id_from);
```
Comment (Quy) [inline administrator/components/com_messages/Table/MessageTable.php:59]: Add `use Joomla\CMS\User\User;`

Comment (brianteeman) [reply administrator/components/com_messages/Table/MessageTable.php:59]: done

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks for merging and testing

**Pull request merged**